build/lib/test/test.js

```diff
@@ -341,6 +342,154 @@ async function killiOSSimulator() {
 	});
 }
 
+class DeviceTestDetails {
+	/**
+	 *
+	 * @param {string} name device/emulator name (as reported via log output. Defaults to empty string)
+	 * @param {'emulator'|'device'|'simulator'} target
```

Comment:
<!--
  0 failure: 
  1 warning:  build/lib/test/te...
  
  
  DangerID: danger-id-Danger;
  File: build/lib/test/test.js;
  Line: 349;
-->

- :warning: build/lib/test/test.js line 349 – Missing JSDoc parameter description for 'target'. (valid-jsdoc)